blocks/init/src/Blocks/custom/tabs/components/tabs-panel.js

```diff
@@ -0,0 +1,39 @@
+import { selector } from '@eightshift/frontend-libs/scripts/helpers';
+import React from 'react';
+import manifest from './../manifest.json';
+
+export const TabsPanel = (props) => {
+
+    const { childBlocks, activeTab, setAttributes } = props;
+    const tabItems = childBlocks.map((block) => 
+        {
+            let classes = selector(manifest.tabPanelItemElement, manifest.blockClass, manifest.tabPanelItemElement);
```

Comment:
These properties can be extracted out of manifest like in the other files, for readability sake.